tensorrt_llm/_bootstrap.py

```
@@ -0,0 +1,176 @@
+
+    # Load FT decoder layer and torch custom ops.
+    if platform.system() == "Windows":
+        ft_decoder_lib = project_dir + "/libs/th_common.dll"
+    else:
+        ft_decoder_lib = project_dir + "/libs/libth_common.so"
+    try:
+        torch.classes.load_library(ft_decoder_lib)
+        from ._torch.custom_ops import _register_fake
+
+        _register_fake()
+    except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)